x/supplier/keeper/msg_server_create_claim.go

```diff
@@ -45,11 +55,22 @@ func (k msgServer) CreateClaim(goCtx context.Context, msg *suppliertypes.MsgCrea
 	if !found {
 		return nil, suppliertypes.ErrSupplierNotFound.Wrapf(
 			"supplier address %q in session ID %q",
-			msg.SupplierAddress,
+			msg.GetSupplierAddress(),
 			sessionRes.GetSession().GetSessionId(),
 		)
 	}
 
+	// TODO_TECHDEBT(#181): refactor once structured logging is available.
```

Comment:
Just a heads up that it'll be a couple rounds (because others have PRs in flight as well)